libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts

```diff
@@ -931,4 +938,51 @@ describe('DropdownComponent', () => {
       });
     });
   });
+
+  describe('when inside host component with ChangeDetectionStrategy.OnPush', () => {
+    let spectator: SpectatorHost<DropdownComponent>;
+    let buttonElement: HTMLButtonElement;
+    let cardElement: HTMLElement;
+
+    const createHost = createHostFactory({
+      component: DropdownComponent,
+      declarations: [
+        MockComponents(ButtonComponent, CardComponent, ItemComponent, IconComponent, IonItem),
+      ],
+      host: OnPushHostComponent,
+    });
+
+    beforeEach(() => {
+      spectator = createHost(`<kirby-dropdown></kirby-dropdown>`, {
+        props: {
+          items: items,
+        },
+      });
+      buttonElement = spectator.query('button[kirby-button]');
+    });
+
+    beforeEach(fakeAsync(() => {
+      cardElement = spectator.query('kirby-card');
+      // Assert that card is initially hidden:
+      expect(cardElement).toBeHidden();
+      expect(cardElement).toHaveComputedStyle({ opacity: '0' });
+      // Act:
+      spectator.click('button');
```

Comment:
Using `spectator.click` is a false positive, since it invokes `detectChanges` - have a look here: https://github.com/ngneat/spectator/blob/master/projects/spectator/src/lib/base/dom-spectator.ts#L126